data/src/main/java/com/example/data/repo/remote/RemoteSource.java

```diff
@@ -0,0 +1,23 @@
+package com.example.data.repo.remote;
+
+import com.example.data.dtos.ArticleListDto;
+import com.example.data.repo.remote.NewsAPI;
+
+import java.util.Locale;
+
+import io.reactivex.Single;
+import io.reactivex.schedulers.Schedulers;
+
+public class RemoteSource {
```

Comment:
let's rename this class, our local class is called NewsLocalDataStore, for consistency let's rename this one to NewsRemoteDataStore (naming conventions are part of an architecture)